backend/swift/swift.go

```diff
@@ -223,6 +223,8 @@ type Options struct {
 	ChunkSize                   fs.SizeSuffix        `config:"chunk_size"`
 	NoChunk                     bool                 `config:"no_chunk"`
 	Enc                         encoder.MultiEncoder `config:"encoding"`
+	FetchUntilEmptyPage         bool                 `config:"fetch_until_empty_page"`
```

Comment:
This is missing the definition of `fetch_until_empty_page` - it needs to go in the config section above.